zone/raids.cpp

```diff
@@ -1653,9 +1685,12 @@ void Raid::GetRaidDetails()
 
 	auto row = results.begin();
 
-	locked = Strings::ToInt(row[0]);
-	LootType = Strings::ToInt(row[1]);
-	motd = std::string(row[2]);
+	locked        = Strings::ToInt(row[0]);
+	LootType      = Strings::ToInt(row[1]);
+	motd          = std::string(row[2]);
+	MarkedNPCs[0] = Strings::ToUnsignedInt(row[3]);
```

Comment:
`MarkedNPCs` should be `snake_case` eg `marked_npcs`

Reply:
Updated